### PR TITLE
Eitanya/bump envoy 1.26.x

### PR DESCRIPTION
**Pull request by EItanya**

### CODE REVIEW

File: ci/do_ci.sh [hunk 1]
```diff
@@ -4,17 +4,27 @@ bazel fetch //source/exe:envoy-static
 
 SOURCE_DIR="$(bazel info workspace)"
 
+git clone https://github.com/envoyproxy/envoy.git /tmp/envoy
```
Comment (nfuden) [inline ci/do_ci.sh:7]:
```suggestion
# will be reverted or updated in https://github.com/solo-io/solo-projects/issues/5098
git clone https://github.com/envoyproxy/envoy.git /tmp/envoy
```

Comment (nfuden) [inline ci/do_ci.sh:7]:
```suggestion
# will be reverted or updated in https://github.com/solo-io/solo-projects/issues/5098
git clone https://github.com/envoyproxy/envoy.git /tmp/envoy
```

File: ci/do_ci.sh [hunk 2]
```diff
@@ -4,17 +4,28 @@ bazel fetch //source/exe:envoy-static
 
 SOURCE_DIR="$(bazel info workspace)"
 
+# will be reverted or updated in https://github.com/solo-io/solo-projects/issues/5098
```
Comment (nfuden) [inline ci/do_ci.sh:7]:
```suggestion
# will be reverted or updated in https://github.com/solo-io/envoy-gloo/issues/246
```

**Review by nfuden — APPROVED** [review]

File: bazel/foreign_cc/dynamic-metadata-matchingdata-25.patch [hunk 3]
```diff
@@ -1,341 +0,0 @@
-diff --git a/envoy/network/filter.h b/envoy/network/filter.h
```
Comment (kdorosh) [inline bazel/foreign_cc/dynamic-metadata-matchingdata-25.patch:1]: cc @ashishb-solo was this needed for deprecated cipher passthrough?

File: bazel/foreign_cc/filter-state-matching-input-25.patch [hunk 4]
```diff
@@ -1,800 +0,0 @@
-diff --git a/api/envoy/extensions/matching/common_inputs/network/v3/network_inputs.proto b/api/envoy/extensions/matching/common_inputs/network/v3/network_inputs.proto
```
Comment (kdorosh) [inline bazel/foreign_cc/filter-state-matching-input-25.patch:1]: cc @ashishb-solo was this needed for deprecated cipher passthrough?

File: bazel/repositories.bzl [hunk 5]
```diff
@@ -70,9 +70,6 @@ def envoy_gloo_dependencies():
     # passthrough and only need to be backported onto envoy v1.25.x
     # these should be removed when moving to v1.26.x since this code exists in
     # upstream at that point.
-    _repository_impl("envoy", patches = [
-            "@envoy_gloo//bazel/foreign_cc:filter-state-matching-input-25.patch", # https://github.com/envoyproxy/envoy/pull/25856
-            "@envoy_gloo//bazel/foreign_cc:dynamic-metadata-matchingdata-25.patch", # https://github.com/envoyproxy/envoy/pull/26311
-        ])
+    _repository_impl("envoy")
```
Comment (kdorosh) [inline bazel/repositories.bzl:73]: @ashishb-solo nevermind, i see that the patches made it into upstream :muscle:

File: bazel/foreign_cc/dynamic-metadata-matchingdata-25.patch (hunk 3 above)
Comment (kdorosh) [reply bazel/foreign_cc/dynamic-metadata-matchingdata-25.patch:1]: nevermind https://github.com/solo-io/envoy-gloo/pull/244/files#r1228180660

**Comment by kdorosh** [comment]
does this supersede https://github.com/solo-io/envoy-gloo/pull/203 ? should close that PR?

**Comment by EItanya** [comment]
Yes

**Pull request merged**